api/users/serializers.py

```diff
@@ -1,6 +1,35 @@
 from rest_framework import serializers as ser
-
 from api.base.serializers import JSONAPISerializer, LinksField, Link
+from website.models import User
+from rest_framework.parsers import JSONParser
+
+
+class JobsSerializer(ser.ListField):
+    startYear = ser.CharField(allow_blank=True, allow_null=True)
+    title = ser.CharField()
+    startMonth = ser.IntegerField(allow_null=True)
+    endMonth = ser.IntegerField(max_value=12, min_value=1, allow_null=True)
+    endYear = ser.CharField(allow_blank=True, allow_null=True)
+    ongoing = ser.BooleanField()
+    department = ser.CharField()
+    institution = ser.CharField()
+
+    class Meta:
+        type_ = 'jobs'
+
+
+class SchoolsSerializer(ser.ListField):
```

Comment:
This serializer is almost exactly the same as the jobs serializer. DRY this up a bit.